core/src/commonMain/kotlin/com/ustadmobile/core/controller/ReportTemplatePresenter.kt

```diff
@@ -0,0 +1,49 @@
+package com.ustadmobile.core.controller
+
+import com.ustadmobile.core.db.dao.ReportDao
+import com.ustadmobile.core.util.ext.toQueryLikeParam
+import com.ustadmobile.core.util.safeStringify
+import com.ustadmobile.core.view.ReportEditView
+import com.ustadmobile.core.view.ReportTemplateView
+import com.ustadmobile.core.view.UstadEditView
+import com.ustadmobile.door.DoorLifecycleOwner
+import com.ustadmobile.lib.db.entities.Report
+import com.ustadmobile.lib.db.entities.UmAccount
+import org.kodein.di.DI
+
+class ReportTemplatePresenter(context: Any, arguments: Map<String, String>, view: ReportTemplateView,
+                              di: DI, lifecycleOwner: DoorLifecycleOwner)
+    : UstadListPresenter<ReportTemplateView, Report>(context, arguments, view, di, lifecycleOwner) {
+
+    override fun onCreate(savedState: Map<String, String>?) {
+        super.onCreate(savedState)
+        updateListOnView()
+    }
+
+    override suspend fun onCheckAddPermission(account: UmAccount?): Boolean {
+        return true
```

Comment:
Isn't this always false? There isn't really just a plus button here. The user has to go the report they want, and then click save as a template.